playback/interception/files/file_interception.py

```diff
@@ -129,7 +129,7 @@ def _serialize_file(content, file_path):
         :rtype: dict[str, str]
         """
 
-        if sys.version_info.major == 2:
+        if six.PY2:
```

Comment:
I now see you have introduce pep8 error here in line 133, 135 in previous change
We really need to add linter here

Reply:
on it